### DISCUSSION TITLE
how to update config.json when my design with two clock input with different frequence?

### DISCUSSION THREAD

**Discussion by TonyHo722** [The-OpenROAD-Project/OpenLane#2024] [Q&A] [answered]
@kareefardi 
in https://github.com/The-OpenROAD-Project/OpenLane/blob/master/docs/source/reference/configuration.md, the CLOCK_PORT is for "The name of the design's clock port".

how to update config.json when my design with two clock input with different frequence?
any example for it?
Thanks

**Comment by salmen20** [comment]
Hello , have you found any solution for this ?
Thanks.

**Reply by TonyHo722** [reply, marked as answer]
you can reference my commit in this link https://github.com/TonyHo722/caravel-lab/commit/8598bf0d0da517728e428a85cb4e9fcdd148bda4
I update base.sdc and create clock for wb_clk_i.
